### PR TITLE
fix: paginated changelog

### PR DESCRIPTION
**Pull request by abvthecity**
Implements pagination on the long-scrolling changelog page

### CODE REVIEW

File: packages/ui/app/src/components/BottomNavigationNeighbors.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { DocsContent } from "../resolver/DocsContent";
+import { BottomNavigationButtons } from "./BottomNavigationButtons";
+
+interface BottomNavigationNeighborsProps {
+    neighbors?: DocsContent.Neighbors;
+}
+
+export const BottomNavigationNeighbors: FC<BottomNavigationNeighborsProps> = (props) => {
+    const toHref = useToHref();
+    const neighborsFromAtom = useAtomValue(NEIGHBORS_ATOM);
```
Comment (RohinBhargava) [inline packages/ui/app/src/components/BottomNavigationNeighbors.tsx:14]: does atom state refresh on new deployments?

Comment (abvthecity) [reply packages/ui/app/src/components/BottomNavigationNeighbors.tsx:14]: the atom is client-side only so it's reset only on the client

Comment (RohinBhargava) [reply packages/ui/app/src/components/BottomNavigationNeighbors.tsx:14]: what happens if there is a changelog entry and the boundary of each page changes? is it ok to have > 10 neighbors in some cases?

Comment (RohinBhargava) [reply packages/ui/app/src/components/BottomNavigationNeighbors.tsx:14]: or am I misinterpreting what this is doing?

**Review by RohinBhargava — APPROVED** [review]

File: packages/ui/app/src/components/BottomNavigationNeighbors.tsx (hunk 1 above)
Comment (abvthecity) [reply packages/ui/app/src/components/BottomNavigationNeighbors.tsx:14]: Screenshot 2024-09-12 at 3 19 09 PM

Comment (abvthecity) [reply packages/ui/app/src/components/BottomNavigationNeighbors.tsx:14]: it just renders the left-right buttons

**Pull request merged**